### PR TITLE
Create 2020-11-01-cassoulet.md

### PR DESCRIPTION
**Pull request by zmagg**

### CODE REVIEW

File: _posts/meat/2020-11-01-cassoulet.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Cassoulet
+category: meat
+tags: recipeish
+---
+
+We've been on a cassoulet kick this summer in the foggy Inner Richmond. The first few times we made it with duck confit pieces. This last time, we had run out of duck confit on hand, but still had some of the [jelly that sinks at the bottom of duck fat)](https://www.reddit.com/r/AskCulinary/comments/6g2klj/the_gelatin_of_duck_fat/) when you render it yourself, so we used that instead.
```
Comment (mookerji) [inline _posts/meat/2020-11-01-cassoulet.md]: > duck confit pieces

duck confit ~pieces~. 'Pieces' here is redundant.

Comment (mookerji) [reply _posts/meat/2020-11-01-cassoulet.md]: I think the 'jelly' here is actually called 'aspic', but maybe calling it that would sound pretentious. 'duck gelatin'?

File: _posts/meat/2020-11-01-cassoulet.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+This recipe is riffing off of a few sources, and I don't really consider it something that I'd tell other people to do (you'll see why in a moment). [Kenji's cassoulet recipe](https://www.seriouseats.com/recipes/2014/10/traditional-french-cassoulet-recipe.html)](https://www.seriouseats.com/recipes/2014/10/traditional-french-cassoulet-recipe.html) is faster end to end, and Elizabeth David's cassoulet recipes in _French Provincial Cooking_ all call for mutton shoulder (!!?!?) which honestly I'm not even sure you can easily buy mutton (older lamb) in the U.S. these days. I also watched the [video](https://www.ranchogordo.com/products/cassoulet-tarbais-bean) on Rancho Gordo's cassoulet bean page. I've wanted to make cassoulet since I first read about it, it's always felt really romantic to me. Now that we've made it a few times, I think it's more delicious than I originally thought, but less romantic, it's basically just sausage and beans.
```
Comment (mookerji) [inline _posts/meat/2020-11-01-cassoulet.md]: > less romantic, it's basically

less romantic: it's basically

File: _posts/meat/2020-11-01-cassoulet.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+Ingredients:
+* Cassoulet beans (1/2 lb or 1 lb dry) from (Rancho Gordo)[https://www.ranchogordo.com/products/cassoulet-tarbais-bean]
```
Comment (mookerji) [inline _posts/meat/2020-11-01-cassoulet.md]: This range of bean volume and sausage below basically double the recipe. The actual amount isn't variable, right? I think we should choose a base volume of beans and build off of that. Also, I know I was all about six sausages for a pound of beans, but man that was just  a lot of sausage.

File: _posts/meat/2020-11-01-cassoulet.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+* Aromatics: 1 celery rib, 1 carrot, 6 bay leaves, black peppercorn
+* 4 ounces guanciale / pancetta 
+* 3-6 pork sausages
+* 1 white onion, diced
+* 2-3 pieces duck confit with a few tablespoons duck fat OR
+* 3 chicken thighs and some duck confit jelly / duck fat jelly / duck fat
+* (optional) Knox gelatin
+
+Recipe:
+* One or two days prior to cooking (lol), soak the beans. Room temperature is fine if your house is <75 degrees, if over, place them in the fridge to soak. If you accidentally end up soaking them too soon, you can always keep them in the soaking water in the fridge for up to 3-4 days.
```
Comment (mookerji) [inline _posts/meat/2020-11-01-cassoulet.md:25]: `< 75F`

File: _posts/meat/2020-11-01-cassoulet.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+* 2-3 pieces duck confit with a few tablespoons duck fat OR
+* 3 chicken thighs and some duck confit jelly / duck fat jelly / duck fat
+* (optional) Knox gelatin
+
+Recipe:
+* One or two days prior to cooking (lol), soak the beans. Room temperature is fine if your house is <75 degrees, if over, place them in the fridge to soak. If you accidentally end up soaking them too soon, you can always keep them in the soaking water in the fridge for up to 3-4 days. 
+* Parcook the beans. Add beans to cold water with the aromatics and a few tablespoons of olive oil / duck fat. Bring water to a boil and then lower to the lowest setting possible. Cook for 3-4 hours. Yes. 3-4 hours. Add salt at the end of cooking, turn off the heat. Store in the fridge if you're not continuing to make cassoulet that day. They can store in their bean broth in the fridge for another 5 days.
+* If using duck fat jelly, add it to the bean broth on the day you're cooking the cassoulet. Taste the resulting 'stock' of bean broth and duck fat and duck jelly. If needed for more body, add additional Knox gelatin packets to the warmed up broth. Resulting 'stock' should taste slightly less salty than you'd like in a soup.
```
Comment (mookerji) [inline _posts/meat/2020-11-01-cassoulet.md:27]: > duck fat jelly

 duck gelatin

File: _posts/meat/2020-11-01-cassoulet.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+* In the dutch oven you plan to cook the cassoulet in, ad guanciale / pancetta pieces (diced very small) to pan with two tablespoons of duck fat. Let the fat render until the pieces get crispy. Remove the pork.
```
Comment (mookerji) [inline _posts/meat/2020-11-01-cassoulet.md:28]: ad => add

File: _posts/meat/2020-11-01-cassoulet.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+* Brown the chicken thighs / duck confit on both sides--add them dry to the pan with some black pepper. It'll take 5 minutes on both sides to brown. Remove chicken.
+* Brown sausages. Remove sausages. 
+* Add diced onion. If there's not enough fat at this point in the pan, add more. Salt onion just a little. Cook until onion is translucent.
+* Add in beans and all the meat and cover it with the bean broth / duck fat jelly combo. Liquid should just barely cover the beans and the meat. If not enough liquid, add more water.
```
Comment (mookerji) [inline _posts/meat/2020-11-01-cassoulet.md]: The meat should be on top. Actually, I think the thighs should submerge the sausage?

**Review by mookerji — COMMENTED** [review]
See comments!

File: _posts/meat/2020-11-01-cassoulet.md (hunk 3 above)
Comment (zmagg) [reply _posts/meat/2020-11-01-cassoulet.md]: lolol

**Pull request merged**